moveit_planners/ompl/ompl_interface/src/planning_context_manager.cpp

```diff
@@ -90,6 +90,11 @@ struct PlanningContextManager::CachedContexts
 };
 
 MultiQueryPlannerAllocator::~MultiQueryPlannerAllocator()
+{
+  storePlannerData();
```

Comment:
It seems like that shouldn't be necessary. This is configurable through the `ompl_planning.yaml` file as described [here](https://moveit.picknik.ai/galactic/doc/examples/ompl_interface/ompl_interface_tutorial.html#persistent-roadmaps).

Reply:
I left this there because it was there before (see [planning_context_manager.cpp#L92-L102](https://github.com/ros-planning/moveit2/blob/main/moveit_planners/ompl/ompl_interface/src/planning_context_manager.cpp#L92-L102)). I would also argue that storing something while deconstructing an object is not a good idea.